### DISCUSSION TITLE
virtualbox headless frontend占用巨高

### DISCUSSION THREAD

**Discussion by qq994376571** [MaaAssistantArknights/MaaAssistantArknights#8970] [General]
截图时间一开始都正常，没一会就跑到1000ms，一看virtualbox headless frontend把cpu占满了。这应该不是性能问题，不开maa的话virtualbox headless frontend 占用很低20-30%，再加上重装系统前都正常，也是win10系统。所以这是什么原因导致的？

**Comment by qq994376571** [comment]
开启maa后这个占用随着每次战斗而上升，一开始只有20-30，等第二次战斗就翻倍，最后就卡死了

**Comment by qq994376571** [comment]
用的雷电9，然后不仅是战斗，公招等都会导致占用升高不降

**Comment by status102** [comment]
Duplicate

**Reply by qq994376571** [reply]
以前没有这个问题，这个和什么有关系

**Reply by ABA2396** [reply]
谁卡找谁啊，这不是模拟器处理不好截图吗

**Reply by qq994376571** [reply]
> 谁卡找谁啊，这不是模拟器处理不好截图吗

问题是 之前一直能用，除非说再试试换模拟器

**Reply by ABA2396** [reply]
那你下个老版本的maa试试）

**Reply by qq994376571** [reply]
> ）

鉴定完毕，是雷神的问题，换成mumu就好了

**Reply by status102** [reply]
- link to #8971
